README.md

```diff
@@ -18,6 +18,63 @@ To install [the current release](https://pypi.org/project/nvflare), you can simp
+ in a similar way to how it would run in a real deployment. Researchers can quickly 
+ build out new components and jobs that can then be directly used in a real production deployment.
+
+* [FLARE Dashboard](https://nvflare.readthedocs.io/en/main/user_guide/dashboard_api.html) --
+  NVFLARE's web UI. In its initial incarnation, the Flare Dashboard is used to help
+  project setup, user registration, startup kits distribution and dynamic provisions.  
+
+* [Site-policy management](https://nvflare.readthedocs.io/en/main/user_guide/site_policy_management.html) -- 
+  Prior to NVFLARE 2.2, all policies (resource management, authorization and privacy protection, logging configurations) 
+  can only be defined by the Project Admin during provision time; and authorization policies are centrally enforced by the FL Server.
+  NVFLARE 2.2 makes it possible for each site to define its own policies in the following areas:
+  * Resource Management: the configuration of system resources that are solely the decisions of local IT.
+  * Authorization Policy: local authorization policy that determines what a user can or cannot do on the local site. see related [Federated Authorization](https://nvflare.readthedocs.io/en/main/user_guide/federated_authorization.html)
+  * Privacy Policy: local policy that specifies what types of studies are allowed and how to add privacy protection to the learning results produced by the FL client on the local site.
+  * Logging Configuration: each site can now define its own logging configuration for system generated log messages.
+  
+* [Federated XGBoost](<https://github.com/NVIDIA/NVFlare/tree/main/examples/xgboost>) --
+  We developed federated XGBoost for data scientists to perform traditional machine learning. In this release, we provide several 
+  approaches for the horizontal federated XGBoost algorithms. 
+  * Histogram-based Collaboration -- leverages newly added XGBoost open-source distributed XGBoost training algorithm to operate in a federated manner.
+  * Tree-based Collaboration -- individual trees are independently trained on each client's local data without aggregating the global sample gradient histogram information. 
+  Trained trees are collected and passed to the server / other clients for aggregation and further boosting rounds.
+  
+* [Federated Statistics](<https://github.com/NVIDIA/NVFlare/tree/main/examples/federated_statistics>) -- 
+  built-in federated statistics operators that can generate global statistics based on local client side statistics. 
+  The results, for all features of all datasets at all sites as well as global aggregates, can be visualized via the visualization utility in the notebook.  
+
+* [MONAI Integration](<https://github.com/NVIDIA/NVFlare/tree/main/integration/monai/README.md>)
+  In 2.2 release, we provided two implementations by leveraging Monai "bundle"
+  * Monai ClientAlgo Integration -- enable Monai bundle directly run on NVFLARE federated setting
```

Comment:
"enable Monai bundle directly run on NVFLARE federated setting" -> "enable running MONAI bundles directly in a federated setting using NVFLARE"